angular.json

```diff
@@ -7,10 +7,114 @@
 			"root": "",
 			"sourceRoot": "src",
 			"projectType": "application",
-			"prefix": "terra",
+			"prefix": "tc",
 			"schematics": {
 				"@schematics/angular:component": {
-					"styleext": "scss"
+					"style": "scss"
+				}
+			},
+			"architect": {
+				"build": {
+					"builder": "@angular-devkit/build-angular:browser",
+					"options": {
+						"outputPath": "dist",
+						"index": "src/index.html",
+						"main": "src/main.ts",
+						"polyfills": "src/polyfills.ts",
+						"tsConfig": "src/tsconfig.app.json",
+						"assets": [
+							"src/assets"
+						],
+						"styles": [
+							"node_modules/bootstrap/scss/bootstrap.scss",
```

Comment:
in der *src/styles.scss* wird das ebenfalls importiert. Also entweder dort oder hier raus. Je nachdem ob es auch im css bundle der lib mit drin sein soll.

Reply:
Übernehme ich

Reply:
https://github.com/plentymarkets/terra-components/pull/1028